### ISSUE TITLE
Make the index.html to be more responsive website

### ISSUE THREAD

**Issue by nan2iz** [CS580-Thunderbird/study-hard-play-hard#47]
Apply bootstrap or other techniques to make our main webpage to be more responsive to all devices, e.g. mobile, smaller laptop, bigger laptop.

**Comment by nan2iz** [comment]
It's not look nice after you applied it. Please recheck again.

**Comment by kushal-patel** [comment]
Yeah, it looks like there is a margin on the right side of the toolbar. I will look into it. Thanks.
